packages/office-addin-dev-certs/src/verify.ts

```diff
@@ -9,13 +9,15 @@ import * as defaults from "./defaults";
 import { usageDataObject } from "./defaults";
 import { ExpectedError } from "office-addin-usage-data";
 
+const powershellOutputMarker = `[${new Date().getTime()}]`
```

Comment:
This should be for Win32 only. Can you move it before defaultCommand variable below which is inside case "win32"?

Reply:
The value should be unique and is used later on in `isCaCertificateInstalled` to check against the actual result string. 
Not sure how to rewrite this and keeping it readable. I could use a generator function (see below), but that seems way less readable. Please advise.

```js
function* generateOutputMarker() {
  const value = `[${new Date().getTime()}]`;
  while (true) {
    yield value;
  }
}

// Usage in `getVerifyCommand` and `isCaCertificateInstalled`
const outputMarker = generateOutputMarker().next().value
```

Reply:
Please let me know if you disagree with creating anything in that scope that relates to a single platform or just with the variable being created even when we might not use it.

This is simpler still, since `crypto` is already in here:

```js
function getOutputMarker() {
    return `[${crypto.createHash("md5").update(JSON.stringify(defaults).join('')).digest("hex")}]`
}
```
This would then only be called in the `win32` paths in  `getVerifyCommand` and `isCaCertificateInstalled` like so:
```js
const outputMarker  = getOutputMarker()
```

Simply putting this in two places seems needlessly fragile:
```js
const outputMarker  = `[${crypto.createHash("md5").update(JSON.stringify(defaults).join('')).digest("hex")}]`
```